lib/fernet/verifier.rb

```diff
@@ -61,11 +64,17 @@ def enforce_ttl=(val)
 
   private
     def create_token!
+      secrets = Array(@opts.fetch(:secret)) + @opts.fetch(:additional_secrets, [])
+      secret = secrets.find do |secret|
```

Comment:
Can we skip this iteration if no additional secrets were provided? That way, you don't have to take a performance hit if you only have one secret.